pages/common/ps.md

```diff
@@ -11,9 +11,9 @@
 
 `ps auxww`
 
-- Search for a process that matches a string:
+- Search for a process that matches a string (the brackets filter out `grep` itself):
```

Comment:
```suggestion
- Search for a process that matches a string (the brackets will prevent `grep` from matching itself):
```

This phrasing makes it more clear that the brackets are `grep` specific and not `ps` specific.